packages/oasgraph/src/types/options.ts

```diff
@@ -89,6 +92,13 @@ export type Options = {
    * operationId. 
    */
   operationIdFieldNames?: boolean
+
+  /**
+   * Allow to override or add options to the node's request object used to make
```

Comment:
I am in favor of changing "Allow" into "Allows" to match the tense of the other comments. What do you think?

Reply:
+1